apps/swirl-docs/src/lib/tokens/src/token.model.ts

```diff
@@ -1,10 +1,68 @@
 export type Token = {
   name: string;
-  type: "color" | "size";
+  type:
```

Comment:
**nitpick:** Could we introduce ColorTokenCategory and SizeTokenCategory, even if it is only a simple string type?

Reply:
yeah good idea